### PR TITLE
Issue 85 fix

### PR DESCRIPTION
**Pull request by rashmi73**

### CODE REVIEW

**Comment by rashmi73** [comment]
@ThomasKranitsas doing

File: src/services/ChallengeService.js [hunk 1]
```diff
@@ -284,10 +287,26 @@ searchChallenges.schema = {
...
     createdBy: Joi.string(),
     updatedBy: Joi.string(),
-    memberId: Joi.number().integer().positive()
+    memberId: Joi.number().integer().positive(),
+    sortBy: Joi.string()
   })
 }
 
+
+async function validateSortQueryFields (criteria) {
```
Comment (ThomasKranitsas) [inline src/services/ChallengeService.js:296]: why didn't you just use `Joi` for validation? The valid values should be in the app-constants.js and you should use the `.valid()` method from Joi for validation.

Comment (rashmi73) [reply src/services/ChallengeService.js:296]: I saw a function which validated fields in similar way hence to maintain consistency I did this. will change it as per your comment

Comment (rashmi73) [reply src/services/ChallengeService.js]: @ThomasKranitsas change has been pushed. Kindly review

**Pull request merged**